easybuild/framework/easyconfig/format/yeb.py

```diff
@@ -0,0 +1,165 @@
+# #
+# Copyright 2013-2015 Ghent University
+#
+# This file is part of EasyBuild,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://vscentrum.be/nl/en),
+# the Hercules foundation (http://www.herculesstichting.be/in_English)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# http://github.com/hpcugent/easybuild
+#
+# EasyBuild is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v2.
+#
+# EasyBuild is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+# #
+"""
+This describes the easyconfig YAML format
+@author: Caroline De Brouwer (Ghent University)
+"""
+
+import os
+import re
+import tempfile
+import yaml
+from vsc.utils import fancylogger
+
+from easybuild.framework.easyconfig.format.format import FORMAT_DEFAULT_VERSION, get_format_version
+from easybuild.framework.easyconfig.format.format import EasyConfigFormat
+from easybuild.framework.easyconfig.format.version import EasyVersion
+from easybuild.tools.build_log import EasyBuildError, print_msg
+from easybuild.tools.filetools import read_file, write_file
+
+
+_log = fancylogger.getLogger('easyconfig.format.yeb', fname=False)
+
+
+class FormatYeb(EasyConfigFormat):
+    """Support for easyconfig YAML format"""
+    USABLE = True
+
+    def __init__(self, filename):
+        self.filename = filename
+        super(FormatYeb, self).__init__()
+
+    def validate(self):
+        """Format validation"""
+        # TODO for YAML
```

Comment:
just return `True` for now, add a `log.info` statement making it clear that there's no real validation (yet)?